policyengine_us/parameters/gov/states/ne/tax/income/credits/ctc/non_refundable/cap.yaml

```diff
@@ -0,0 +1,12 @@
+description: Nebraska caps the non-refundable child tax credit at this amount.
```

Comment:
i think this credit is for creating businesses, not personal childcare expenses (the $100k limit is a clue; no CTC would get that large). we can skip if so.